centreon/www/class/centreonStatistics.class.php

```diff
@@ -245,6 +245,40 @@ public function getNewAuthenticationOptions()
                         'auto_import' => (bool)$customConfiguration['auto_import']
                     ];
                     break;
+                case 'saml':
+                    $authenticationConditions = $customConfiguration['authentication_conditions'];
+                    $groupsMapping = $customConfiguration['groups_mapping'];
+                    $rolesMapping = $customConfiguration['roles_mapping'];
+                    $data['saml'] = [
+                        'is_forced' => (bool)$row['is_forced'],
+                        'authenticationConditions' => [
+                            'is_enabled' => (bool)$authenticationConditions['is_enabled'],
+                            'trusted_client_addresses'
+                            => count($authenticationConditions['trusted_client_addresses'] ?? []),
+                            'blacklist_client_addresses'
+                            => count($authenticationConditions['blacklist_client_addresses'] ?? []),
+                            'authorized_values' => count($authenticationConditions['authorized_values'] ?? [])
+                        ],
+                        'groups_mapping' => [
+                            'is_enabled' => (bool)$groupsMapping['is_enabled'],
+                            'relations' => $cgRelations
```

Comment:
Previous SQL query is still valid and can make a difference between OIDC and SAML configuration for $cgRelations value?